code/style.css

```diff
@@ -1,58 +1,377 @@
-body {
+* {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  box-sizing: border-box;
+}
+
+/*** GENERAL ***/
+
+body {
+  font-family: 'Courier New', Courier, monospace;
```

Comment:
I see that font-family is already put inside the previous block. We can remove it from the body styling.